core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -346,7 +360,7 @@ object AppType {
     appType match {
       case Galaxy                     => FormattedBy.Galaxy
       case Custom                     => FormattedBy.Custom
-      case RStudio                    => FormattedBy.RStudio
+      case Allowed                    => FormattedBy.RStudio
```

Comment:
Wonder what to do about disk support. Do we want a disk formatted by Allowed-rstudio to be able to be used by Allowed-sas?